deps.edn

```diff
@@ -73,7 +73,7 @@
   info.sunng/ring-jetty9-adapter            {:mvn/version "0.22.3"}             ; Drop-in replacement for official Ring Jetty adapter. Supports Jetty 11 webserver.
   instaparse/instaparse                     {:mvn/version "1.4.12"}             ; Make your own parser
   clj-commons/clj-yaml                      {:mvn/version "1.0.27"}             ; Clojure wrapper for YAML library SnakeYAML
-  io.github.camsaul/toucan2                 {:mvn/version "1.0.524"}
+  io.github.camsaul/toucan2                 {:mvn/version "1.0.535"}
```

Comment:
Minor Toucan 2 version bump. No big changes but it has a few bugfixes and simplifies some of the interfaces we need to implement